backend/src/recipe/recipe.service.ts

```diff
@@ -9,14 +9,17 @@ const recipeRepository = AppDataSource.getRepository(Recipe);
 
 // The lambda saves the recipe first and than uses it's uuid to save the recipe ingredients
 const createRecipe = async (recipeDetails: RecipeDetails) => {
-  const { name, steps, chefUuid, ingredients } = recipeDetails;
+  const { name, steps, chefUuid, ingredients, description, imageUrl } =
```

Comment:
אפשר להוציא מכאן את מה שאתה משתמש ובשאר פשוט ...rest ואז לשים את זה למטה בשמירה במקום לכתוב כל פרמטר בנפרד
```suggestion
  const { name, chefUuid, ingredients, ...rest } =
```